config/gtasa/constants.xml

```diff
@@ -266,21 +308,18 @@
       <Constant Name="STATUS_WRECKED"/>
       <Constant Name="STATUS_TRAIN_MOVING"/>
       <Constant Name="STATUS_TRAIN_NOT_MOVING"/>
-      <Constant Name="STATUS_HELI"/>
-      <Constant Name="STATUS_PLANE"/>
       <Constant Name="STATUS_PLAYER_REMOTE"/>
       <Constant Name="STATUS_PLAYER_DISABLED"/>
       <Constant Name="STATUS_TRAILER"/>
       <Constant Name="STATUS_SIMPLE_TRAILER"/>
```

Comment:
ditto (or maybe not?)

Reply:
STATUS_HELI and STATUS_PLANE doesn't exist in this enum
![image](https://github.com/user-attachments/assets/3b0cacd4-ac6b-4f93-941e-241a7ca1d77a)